libflux/flux-core/src/doc/mod.rs

```diff
@@ -1294,6 +1299,77 @@ mod test {
         );
     }
     #[test]
+    fn test_value_doc_multiline_headline_no_desc() {
```

Comment:
this test passed without changes to the parser, but I wanted to validate that the normal headline didn't have the same problem as the headline for a parameter within a list item.